modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json

```diff
@@ -4906,6 +4906,829 @@
+							"name": "tableCellPaddingY",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-font-size)",
+							"label": "table-font-size",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-font-size"
+								}
+							],
+							"name": "tableFontSize",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--font-weight-normal)",
+							"label": "table-font-weight",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-font-weight"
+								}
+							],
+							"name": "tableFontWeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "1.5rem",
+							"label": "table-header-cell-min-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-header-cell-min-height"
+								}
+							],
+							"name": "tableHeaderCellMinHeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-3)",
+							"label": "table-header-cell-padding-x",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-header-cell-padding-x"
+								}
+							],
+							"name": "tableHeaderCellPaddingX",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-3)",
+							"label": "table-header-cell-padding-y",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-header-cell-padding-y"
+								}
+							],
+							"name": "tableHeaderCellPaddingY",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--table-font-size)",
+							"label": "table-header-font-size",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-header-font-size"
+								}
+							],
+							"name": "tableHeaderFontSize",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--font-weight-semi-bold)",
+							"label": "table-header-font-weight",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-header-font-weight"
+								}
+							],
+							"name": "tableHeaderFontWeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-line-height)",
+							"label": "table-header-line-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-header-line-height"
+								}
+							],
+							"name": "tableHeaderLineHeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-line-height)",
+							"label": "table-line-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-line-height"
+								}
+							],
+							"name": "tableLineHeight",
+							"type": "String"
+						}
+					],
+					"label": "table",
+					"name": "table"
+				},
+				{
+					"frontendTokens": [
+						{
+							"defaultValue": "1.5rem",
```

Comment:
They are not changing. Remove all of them?

Reply:
No, we shouldn't remove them. We need to know why it's not changing them. If we know why, we will know either how to fix it or if it could be removed.

Reply:
Regarding this fix, I tested again today and it works properly